### PR TITLE
Update Validatrium enpoints

### PR DESCRIPTION
**Pull request by Validatrium**

### CODE REVIEW

File: archway-1/chain.json [hunk 1]
```diff
@@ -241,6 +254,10 @@
       {
         "address": "archway-grpc.openbitlab.com:443",
         "provider": "openbitlab"
+      },
+      {
+        "address": "https://grpc-archway.mainnet.validatrium.club",
```
Comment (jlehtimaki) [inline archway-1/chain.json:259]: This URL does  answer

**Review by jlehtimaki — CHANGES_REQUESTED** [review]

File: archway-1/chain.json (hunk 1 above)
Comment (Validatrium) [reply archway-1/chain.json]: Upgraded grpc link

Comment (jlehtimaki) [reply archway-1/chain.json]: Still getting:
```
grpcurl grpc-archway.mainnet.validatrium.club:443 list
Failed to dial target host "grpc-archway.mainnet.validatrium.club:443": dial tcp 135.181.58.28:443: connect: connection refused
```

Comment (jlehtimaki) [reply archway-1/chain.json]: and the conflicts need to be fixed as well :)

Comment (Validatrium) [reply archway-1/chain.json]: Conflicts are resolved

Comment (Validatrium) [reply archway-1/chain.json]: > Still getting:
> 
> ```
> grpcurl grpc-archway.mainnet.validatrium.club:443 list
> Failed to dial target host "grpc-archway.mainnet.validatrium.club:443": dial tcp 135.181.58.28:443: connect: connection refused
> ```
```bash
grpcurl -plaintext grpc-archway.mainnet.validatrium.club:27490 list
archway.rewards.v1.Query
archway.tracking.v1.Query
...
```

**Review by shahbazn — APPROVED** [review]
lgtm

**Pull request merged**